app/src/main/java/edu/wpi/first/shuffleboard/app/PluginPaneController.java

```diff
@@ -0,0 +1,82 @@
+package edu.wpi.first.shuffleboard.app;
+
+import edu.wpi.first.shuffleboard.api.plugin.Plugin;
+import edu.wpi.first.shuffleboard.app.plugin.PluginLoader;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.jar.JarFile;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javafx.beans.property.ReadOnlyStringWrapper;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.CheckBoxTableCell;
+import javafx.scene.layout.Pane;
+import javafx.stage.FileChooser;
+
+public class PluginPaneController {
+
+  private static final Logger log = Logger.getLogger(PluginPaneController.class.getName());
+
+  @FXML
+  private Pane root;
+  @FXML
+  private TableView<Plugin> pluginTable;
+
+  private TableColumn<Plugin, String> nameColumn = new TableColumn<>("Plugin");
+  private TableColumn<Plugin, Boolean> loadedColumn = new TableColumn<>("Loaded");
+
+  @FXML
+  private void initialize() {
+    nameColumn.setCellValueFactory(param -> new ReadOnlyStringWrapper(param.getValue().getName()));
+    final ObservableList<Plugin> loadedPlugins = PluginLoader.getDefault().getKnownPlugins();
```

Comment:
We'd need to set up a DI framework for that. Maybe later when we're near feature complete